internal/service/onlinearchive/data_source_online_archive.go

```diff
@@ -216,12 +216,12 @@ func schemaOnlineArchive() map[string]*schema.Schema {
 }
 
 func dataSourceMongoDBAtlasOnlineArchiveRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
-	connV2 := meta.(*config.MongoDBClient).AtlasV2
+	connOldV2 := meta.(*config.MongoDBClient).OldAtlasV2
```

Comment:
Just curious if we have a ticket to followup on this so that the resource uses the latest API version.

Reply:
not yet apart from the specific online_archive SDK update ticket . I'm waiting to collect more info (i asked backup team), will do after I have more info